domains/SignUp.js

```diff
@@ -6,6 +6,7 @@ import {
 } from 'react-native';
 import { Formik } from 'formik';
 import * as yup from 'yup';
+// import * as Keychain from 'react-native-keychain';
 import { retrieveSignUpFunction, retrieveSignInFunction } from '../services/parse/auth';
 
```

Comment:
I'll probably approve PLATFORM-123 before this one. Once I merge that one, you can merge master into this PR which will probably change this correct?